pom.xml

```diff
@@ -69,9 +69,9 @@
       <version>${folio-spring-base.version}</version>
     </dependency>
     <dependency>
-      <groupId>org.folio</groupId>
-      <artifactId>folio-query-tool-metadata</artifactId>
-      <version>3.1.0-SNAPSHOT</version>
+	    <groupId>com.github.folio-org</groupId>
+	    <artifactId>folio-query-tool-metadata</artifactId>
+	    <version>fqtm-11-jitpack-SNAPSHOT</version>
```

Comment:
This is temporary, but I wanted sonar to be ok with it. Once ETs are updated I'll do a similar thing in lists so we can get it in rancher